client/bootstrap

```diff
@@ -682,6 +684,86 @@ class Node(object):
         '''
         self.server_.get_resource(url, path)
 
+
+    def create_user(self, user, group, enc_passwd,
+                    root='/persist/local/', ssh_keys=None):
+        '''Create a local user on the bootstrapped node.
+        You can specify:
+         - user: the username
+         - group: the group assigned to the user
+         - enc_passwd: an encrypted password assigned to the user
+         - root: the path where the user's home directory will reside
+         - ssh_keys: public keys that will be copied to ~$HOME/.ssh/authorized_keys
+
+        The method will fail if the user already exists on the node. The
+        $HOME/.ssh directory will be assigned 0700 permissions and the
+        $HOME/.ssh/authorized_keys file will be assigned 0600 permissions in
+        accord with SSH best practices.
+
+        Raises:
+            subprocess.CalledProcessError:
+             -
+            OSError
+             -
+
+        Returns:
+            dict: User details
+
+            Format::
+
+                {'uid':  <USER_ID>,
+                 'gid':  <GROUP_ID>
+                }
+        '''
+        home = os.path.join(root, user)
+        ssh_path = os.path.join(home, '.ssh')
+        auth_path = os.path.join(ssh_path, 'authorized_keys')
+
+        # Create user
+        try:
+            cmd = 'sudo useradd -p %s -m %s -G %s %s' % (enc_passwd, home,
+                                                         group, user)
+            subprocess.call(cmd, stdout=None, stderr=None, shell=True)
```

Comment:
stdout=None, stderr=None are defaults, no need for them. Shouldn't you check if this returns a non-error return code?